src/main/java/ca/gc/aafc/collection/api/entities/CollectingEvent.java

```diff
@@ -82,6 +84,10 @@
       )
   private List<GeoreferenceAssertion> geoReferenceAssertions;
 
+  @OneToOne(fetch = FetchType.LAZY)
+  @ToString.Exclude
+  private GeoreferenceAssertion primaryGeoreferenceAssertion;
```

Comment:
I believe there is  a hibernate annotation to be used @Column, where we can signal this column will be unique.

Reply:
perhaps the OneToOne would handle that?